src/service_images/rails/Gemfile

```diff
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
```

Comment:
Where did you get this gemfile? Was it generated when you initialized rails? Do we need it? Do we need all these gems?